modules/RTC/TraceablePeerConnection.js

```diff
@@ -2053,6 +2061,40 @@ TraceablePeerConnection.prototype.setVideoTransferActive = function(active) {
     return changed;
 };
 
+/**
+ * Sends DTMF tones if possible.
+ *
+ * @param {string} tones - The DTMF tones string as defined by {@code RTCDTMFSender.insertDTMF}, 'tones' argument.
+ * @param {number} duration - The amount of time in milliseconds that each DTMF should last. It's 200ms by default.
+ * @param {number} interToneGap - The length of time in miliseconds to wait between tones. It's 200ms by default.
+ *
+ * @returns {void}
+ */
+TraceablePeerConnection.prototype.sendTones = function(tones, duration = 200, interToneGap = 200) {
+    if (!this._dtmfSender) {
+        if (this.peerconnection.getSenders) {
+            this._dtmfSender = this.peerconnection.getSenders().map(s => s.dtmf)[0];
```

Comment:
Why map over all senders to get the first one?

Reply:
This is the only way I figured to do it in one line. Another way I was trying was to do `find(s => s.dtmf)`, but then you have to do `this._dtmfSender = result.dtmf`. It would be something like this:

```
const rtpSender = this.peerconnection.getSenders().find(s => s.dtmf);
this._dtmfSender = rtpSender.dtmf;
```
Do you prefer it this way or have another idea?

Reply:
Maybe it's more readable this way? WDYT?

Reply:
There's a pretty big difference between find and .map in that case though. With map, if index 0 does not have .dtmf the [0] will still take the undefined, whereas with the find would skip index 0. So that becomes the question, are you looking for the first dtmf in the senders array? Is every sender guaranteed to have dtmf?

Reply:
hmm my observation was that doing map to undefined would kind of remove the item, but maybe I'm wrong will try that again

Reply:
No, you're right, it would map to undefined